examples/fhir2transmart/setup.cfg

```diff
@@ -0,0 +1,21 @@
+[metadata]
+description-file = README.rst
+
+[aliases]
+# Define `python setup.py test`
+test=pytest
+
+[coverage:run]
+branch = True
+source = fhir2transmart
+
+[tool:pytest]
+testpaths = tests
+addopts = --cov --cov-report xml --cov-report term --cov-report html
```

Comment:
should this be "adopts"? I tried running pytest and it doesn't recognize these arguments

Reply:
This was the autogenerated stuff, same as in (../../setup.cfg)[../../setup.cfg], so I expect it to be correct...